pyproject.toml

```
@@ -0,0 +1,13 @@
+[project]
+name = "rrim"
+version = "0.1.0"
+description = ""
+requires-python = ">=3.13"
+dependencies = [
+    "pytest (>=8.3.5,<9.0.0)",
+]
```

1. python 버전이 3.13 이라 호환성 문제가 없을지 걱정되긴 하네요
2. 테스트 의존성은 테스트 그룹에 넣어야 할 것 같아요

---

음 파이썬 버전에 대해서 픽스된 이야기가 없었다보니 일단은 최신버전으로 한거였는데 3.12로 다운그레이드 하는게 좋을까요?

---

아직도 3.13 지원 안하는 라이브러리가 많다보니 혹시나 해서 말씀 드렸습니다

```suggestion
requires-python = ">=3.11,<3.13"
dependencies = [
    "pytest (>=8.3.5,<9.0.0)",
]
```

일단 요렇게 하면 되지 않을까 싶긴 해요 (3.11 이상, 3.13 미만)